bar-trender-front-end/src/components/FreeDashboardQRlist.js

```diff
@@ -13,130 +13,125 @@ import {
   OverlayTrigger,
   Tooltip,
 } from "react-bootstrap";
-class DashboardQRList extends Component{
-  
-    render(){
-      return (
-        <Row>
-          {
-          //Esto se debe convertir en una lista que por cada descuento que tenga 
-          //este local muestre el numero de veces que se ha usado este qr en una tarjeta como esta
-          }
-          
-          <Col lg="3" sm="6">
-            <Card className="card-stats">
-              <Card.Body>
-                <Row>
-                  {/* Por si se quiere añadir imagenes
-                  <Col xs="5">
-                    <div className="icon-big text-center icon-warning">
-                      <i className="nc-icon nc-chart text-warning"></i>
-                    </div>
-                  </Col>
-                  
-                  <Col xs="7">*/}
-                    <div className="numbers">
-                      <p className="card-category">El descuento "2x1 en cervezas" ha sido canjeado por</p>
-                      <Card.Title as="h4">"147" usuarios</Card.Title>
-                    </div>
-                 {// </Col>
-                 }
-                </Row>
-              </Card.Body>
-              {//<Card.Footer>
-              //<hr></hr>
-                //<div className="stats">
-                  //Por si se quiere añadir un boton de acualizar o un boton de de premium
-                  
-                // <i className="fas fa-redo mr-1"></i>
-                //Pasate a premium para mas detalles
-               
-              //  </div>
-              //</Card.Footer>
-            }
-            </Card>
-          </Col>
-          {//A partir de aqui es relleno
-          }
-          <Col lg="3" sm="6">
-            <Card className="card-stats">
-              <Card.Body>
-                <Row>
+import "../views/css/FreeDashboard.css";
+function DashboardQRList(props) {
+  const [modal1, setModal1] = React.useState(false);
+  const [modal2, setModal2] = React.useState(false);
+  const { element } = props;
+  const [appState, setAppState] = useState({
+    discounts: {},
+  });
+
+   useEffect(() => {
+    const apiUrl = "https://develop-backend-sprint-01.herokuapp.com/v1/establishments/"+"1"+"/discounts/get?page=1&all=True";
+    async function loadDiscounts(){
+      await fetch(apiUrl, {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json',
+          'apiKey':'8dDc431125634ef43cD13c388e6eCf11',
+          'token': '',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7173304)
```suggestion
          "token": '',
```